Makefile

```diff
@@ -621,12 +622,24 @@ test-rules: \
 #
 	$(CMD_OPA) test $(REGO_SIGNATURES_DIR) --verbose
 
+
+#
+# types
+#
+
+.PHONY: test-types
+test-types: \
+	.checkver_$(CMD_GO)
+#
```

Comment:
empty line

Reply:
That empty line with # is there in almost all other scripts.

Reply:
Yep, this is following the format I proposed to Makefiles. I'll review this tomorrow morning.

Reply:
ohh didn't know that, my bad :) thanks @rafaeldtinoco